src/Node/HTTP2/Constants.purs

```diff
@@ -0,0 +1,66 @@
+-- | # `http2.constants`
+-- |
+-- | https://nodejs.org/docs/latest/api/http2.html#http2constants
+module Node.HTTP2.Constants where
+
+import Data.Maybe (Maybe(..))
+
+-- | Error codes for `RST_STREAM` and `GOAWAY`.
+-- |
+-- | https://nodejs.org/docs/latest/api/http2.html#http2constants
+type NGHTTP2 = Int
+
+-- | Get the Constant string for an NGHTTP2 error code.
+-- |
+-- | https://nodejs.org/docs/latest/api/http2.html#error-codes-for-rst_stream-and-goaway
+ngHTTP2_String :: NGHTTP2 -> Maybe String
+ngHTTP2_String 0 = Just "NGHTTP2_NO_ERROR"
+ngHTTP2_String 1 = Just "NGHTTP2_PROTOCOL_ERROR"
+ngHTTP2_String 2 = Just "NGHTTP2_INTERNAL_ERROR"
+ngHTTP2_String 3 = Just "NGHTTP2_FLOW_CONTROL_ERROR"
+ngHTTP2_String 4 = Just "NGHTTP2_SETTINGS_TIMEOUT"
+ngHTTP2_String 5 = Just "NGHTTP2_STREAM_CLOSED"
+ngHTTP2_String 6 = Just "NGHTTP2_FRAME_SIZE_ERROR"
+ngHTTP2_String 7 = Just "NGHTTP2_REFUSED_STREAM"
+ngHTTP2_String 8 = Just "NGHTTP2_CANCEL"
+ngHTTP2_String 9 = Just "NGHTTP2_COMPRESSION_ERROR"
+ngHTTP2_String 10 = Just "NGHTTP2_CONNECT_ERROR"
+ngHTTP2_String 11 = Just "NGHTTP2_ENHANCE_YOUR_CALM"
+ngHTTP2_String 12 = Just "NGHTTP2_INADEQUATE_SECURITY"
+ngHTTP2_String 13 = Just "NGHTTP2_HTTP_1_1_REQUIRED"
+ngHTTP2_String _ = Nothing
```

Comment:
Should this be an ADT?

Reply:
I don't think so? Those integer codes come from deep inside Nghttp2 and I think that if we constrain them with the PureScript type system then we’re making type-level promises which we cannot actually guarantee.

Reply:
Sounds good.